# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#goal definition
float64 x_velocity
float64 y_velocity
#float64	omega
